modin/core/storage_formats/pandas/parsers.py

```diff
@@ -634,6 +617,24 @@ def parse(fname, **kwargs):
         ]
 
 
+@doc(_doc_pandas_parser_class, data_type="multiple JSON files simultaneously")
+class PandasJsonGlobParser(PandasJSONParser):
+    @staticmethod
+    @doc(_doc_parse_func, parameters=_doc_parse_parameters_common)
+    def parse(fname, **kwargs):
+        use_json_parser = kwargs.pop("use_json_parser", None)
+        if use_json_parser:
+            return PandasJSONParser.parse(fname, **kwargs)
+
+        warnings.filterwarnings("ignore")
+        df = pandas.read_json(fname, **kwargs)
```

Comment:
isn't it going to read the _whole_ file by pandas? also what is the value of `fname` if I originally called `pd.read_json_glob('foo*.json')`?